scripts/release/mule/package/linux/amd64/docker/package.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo
+date "+build_release begin PACKAGE DOCKER stage %Y%m%d_%H%M%S"
+echo
+
+OS_TYPE="$1"
+ARCH="$2"
+WORKDIR="$3"
+
+if [ -z "$OS_TYPE" ] || [ -z "$ARCH" ] || [ -z "$WORKDIR" ]; then
+    echo "OS=$OS, ARCH=$ARCH and WORKDIR=$WORKDIR variables must be defined."
+    exit 1
+fi
+
+BRANCH=$(./scripts/compute_branch.sh)
+CHANNEL=$(./scripts/compute_branch_channel.sh "$BRANCH")
+TARBALLS="./tmp/node_pkgs/$OS_TYPE/$ARCH"
```

Comment:
Consider renaming this to PACKAGE_ROOT_DIR.

Reply:
Will change.